internal/overlord/servstate/handlers.go

```diff
@@ -437,6 +444,51 @@ func (s *serviceData) exited(waitErr error) error {
 	return nil
 }
 
+// addLastLogs adds the last few lines of service output to the task's log.
+func addLastLogs(task *state.Task, logBuffer *servicelog.RingBuffer) {
+	st := task.State()
+	st.Lock()
+	defer st.Unlock()
+
+	logs, err := getLastLogs(logBuffer)
+	if err != nil {
+		task.Errorf("cannot read service logs: %v", err)
+	}
+	if logs != "" {
+		// Add lastLogLines last lines of service output to the task's log.
+		task.Logf("most recent service output:\n%s", logs)
```

Comment:
s/most/Most/, and s/cannot/Cannot/ above.

Reply:
Done, thanks